.github/workflows/update-gpuci.yml

```diff
@@ -13,12 +13,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Get old cuDF / UCX-Py versions
+      - name: Parse current axis YAML
+        uses: the-coding-turtle/ga-yaml-parser@v0.1.1
+        with:
+          file: ci/axis/dask.yaml
+
+      - name: Get old RAPIDS / UCX-Py versions
         run: |
-          echo OLD_RAPIDS_VER=$(grep -A 1 "RAPIDS_VER" ci/axis/dask.yaml | tail -n1 | grep -o "'.*'" | sed "s/'//g") >> $GITHUB_ENV
-          echo OLD_UCX_PY_VER=$(grep -m1 "UCX_PY_VER" ci/gpuci/run.sh | tail -n1 | grep -o '".*"' | sed 's/"//g') >> $GITHUB_ENV
-          echo RAPIDS_VER=$(grep -A 2 "RAPIDS_VER" ci/axis/dask.yaml | tail -n1 | grep -o "'.*'" | sed "s/'//g") >> $GITHUB_ENV
-          echo UCX_PY_VER=$(grep -m2 "UCX_PY_VER" ci/gpuci/run.sh | tail -n1 | grep -o '".*"' | sed 's/"//g') >> $GITHUB_ENV
+          echo OLD_RAPIDS_VER=$RAPIDS_VER_0 >> $GITHUB_ENV
+          echo OLD_UCX_PY_VER=$(curl -sL https://version.gpuci.io/rapids/$RAPIDS_VER_0) >> $GITHUB_ENV
+          echo RAPIDS_VER=$RAPIDS_VER_1 >> $GITHUB_ENV
+          echo UCX_PY_VER=$(curl -sL https://version.gpuci.io/rapids/$RAPIDS_VER_1) >> $GITHUB_ENV
```

Comment:
From these changes it's not clear to me where do `RAPIDS_VER_0`/`RAPIDS_VER_1` come from. Are those reliable? Is there any chance that 0 and 1 may be swapped elsewhere and cause similar problems in the future?

Reply:
After parsing the axis YAML using this [YAML parsing Action](https://github.com/marketplace/actions/yaml-parser), all of the keys in the YAML get dumped into the GitHub Actions environment, using `_` as a delimiter and appending integers to indicate ordering in lists.

Thus, `RAPIDS_VER_0` / `RAPIDS_VER_1` represent the first and second elements under the `RAPIDS_VER` key in the axis YAML:

https://github.com/rapidsai/dask-build-environment/blob/210c6be6249322f1ac022cd01ab6335c29fa4e2c/ci/axis/dask.yaml#L17-L19

As long as we don't manually change the order of these elements, this ordering should stay consistent.

Reply:
Ok, but this means changes from https://github.com/rapidsai/dask-build-environment/pull/31 are breaking this and must be fixed, right?

Reply:
Yeah, the updates made in #31 are erroneous and my intention after merging this is to close that PR and rerun this updated workflow to make the proper changes